pkg/sfu/buffer/fps.go

```diff
@@ -24,8 +25,9 @@ type FrameRateCalculator interface {
 }
 
 // -----------------------------
-// FrameRateCalculator based on PictureID in VP8
-type FrameRateCalculatorVP8 struct {
+
+// FrameRateCalculator based on PictureID in VPx
+type frameRateCalculatorVPx struct {
```

Comment:
refactoring this to use this for both vp8 and vp9 (used by vp9 when in non-DD mode)